docs/community/SIG-Lifecycle.md

```diff
@@ -0,0 +1,63 @@
+# SIG Lifecycle Guide
+
+This guide describes the lifecycle management for Special Interest Groups (SIGs) in the OCM project, including creation, operation, and archiving/dissolution.
+
+## 1. Creation
+
+- Read the SIG Governance Model and Lifecycle Guide to understand the process and requirements.
+- Draft your SIG charter (see example [Kubernetes SIG Charter](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md)), outlining:
```

Comment:
Do we really need a full charter?

Reply:
do you mean if we need the charter already in the submission for the SIG or a charter for the SIG at all?